tldw_Server_API/app/main.py

```
@@ -5785,6 +5787,13 @@ async def api_metrics():
         app.include_router(chatbooks_router, prefix=f"{API_V1_PREFIX}", tags=["chatbooks"])
     except _IMPORT_EXCEPTIONS as _chatbooks_min_err:
         logger.debug(f"Skipping chatbooks router in minimal test app: {_chatbooks_min_err}")
+    # Sharing endpoints (workspace sharing, tokens, admin)
+    try:
+        from tldw_Server_API.app.api.v1.endpoints.sharing import router as sharing_router
+
+        app.include_router(sharing_router, prefix=f"{API_V1_PREFIX}", tags=["sharing"])
+    except _IMPORT_EXCEPTIONS as _sharing_min_err:
+        logger.debug(f"Skipping sharing router in minimal test app: {_sharing_min_err}")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)